### ISSUE TITLE
Sell on HIGHBB

### ISSUE THREAD

**Issue by Luceid7** [taniman/profit-trailer#1964]
Hello, I'd like to request a feature to sell on LOWBB. Currently, there is no feature to allow for "riding the bands" (riding a trend until price falls below a specific BB threshold). I tried to emulate this using a HIGHBB sell with a sell value limit, but didn't realize that sell value limit was not an option for HIGHBB (which makes sense I was trying to use it in the wrong way). Could this feature be added? Would be very useful, thanks.

**Comment by taniman** [comment]
Please all feature requests should be added on https://github.com/taniman/profit-trailer-enhancements just like the issue template suggested.

In this case I must say that lowbb as a sell strategy does not make any sense. Also once you add your issue on the enhancement branch please make sure to give some more examples

**Comment by Luceid7** [comment]
Good morning Taniman,
I actually did submit my issue on the enhancement site, but it appears that the site was no longer being monitored or checked by PT staff, so I thought that site was no longer active. I apologize for the inconvenience.
And actually, selling on LOWBB makes tons of sense.  Consider the following screenshot:
![example](https://user-images.githubusercontent.com/36316714/42457074-75ac69ac-8364-11e8-8970-41d39c9a64cd.PNG)
Wouldn't you want to sell this coin when it falls below that final trend line, not when it goes above it right at the start? If I try to use HIGHBB to sell, it will sell at the start of the trend as it goes over the HIGHBB line, if I sell at the end when it falls below the trend line, I will have made a lot more money :) This is called" riding the bands" where you ride a trend that is following a bollinger band line, until it falls below it.

**Comment by taniman** [comment]
I see. Interesting. 
sounds good indeed. 

However i will only start looking at the enhancement branch once 2.1.0 has been released as we are not building new features in that version. 

make sure also this explanation with image is there. I will start looking there soon

**Comment by Luceid7** [comment]
Hello again,
I updated the issue I made on the PT enhancement site with this picture and description. Thank you for getting back to me so fast. I really appreciate it. And I totally understand about feature rollouts, and time frames. I just wanted to make you aware of this trading strategy, since I'm guessing it wouldn't require too much coding since 90% of the functionality has already been written into your current BB code.
